### PR TITLE
Adding ability to set desired count for the following services:

### PR DESCRIPTION
**Pull request by nsabri1**
Publisher worker
	Statsd

### CODE REVIEW

**Review by richardTowers — APPROVED** [review]
Looks good to me - consistent with the other applications.

File: terraform/modules/statsd/variables.tf [hunk 1]
```diff
@@ -40,3 +40,9 @@ variable "task_role_arn" {
 variable "vpc_id" {
   type = string
 }
+
+variable "desired_count" {
+  description = "Desired count of Application instances"
```
Comment (sengi) [inline terraform/modules/statsd/variables.tf:45]: I'd recommend omitting the description in this case as it doesn't really add anything.

**Review by sengi — APPROVED** [review]
Cool - thanks for fixing

File: terraform/modules/statsd/variables.tf (hunk 1 above)
Comment (nsabri1) [reply terraform/modules/statsd/variables.tf:45]: sorry missed this comment - will keep that in mind for next

**Pull request merged**